lib/ansible/module_utils/netcfg.py

```diff
@@ -26,12 +26,20 @@
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 import re
+import hashlib
 
 from ansible.module_utils.six.moves import zip
+from ansible.module_utils._text import to_bytes
 from ansible.module_utils.network_common import to_list
 
 DEFAULT_COMMENT_TOKENS = ['#', '!', '/*', '*/', 'echo']
 
+DEFAULT_IGNORE_LINES_RE = set([
```

Comment:
Would a user ever want to change these?
If so maybe do:
`diff_ignore_lines=dict(default=["Using \d+ out of \d+ bytes", "Building configuration", etc]`